autopush/endpoint.py

```diff
@@ -209,13 +209,16 @@ def _write_unauthorized_response(self, message="Invalid authentication",
         self._write_response(401, errno=109, message=message, headers=headers,
                              **kwargs)
 
-    def _response_err(self, fail):
+    def _response_err(self, fail, caller=None):
         """errBack for all exceptions that should be logged
 
         This traps all exceptions to prevent any further callbacks from
         running.
 
         """
+        if not caller:
+            caller = self.__module__
```

Comment:
This is the only source of logged 500,999 errors for this class, so pretty easy to figure out the method. The trick is figuring out the module.  While I don't expect caller to be empty by this point, it may be, so this is a last ditch attempt to harvest the module name.